web-api/openapi/front_office.yaml

```
@@ -174,7 +178,14 @@ paths:
       summary: Semantic search in documents
       description: |-
         Semantically search through the documents in different ways.
-        When a document `id` is used, the system will return documents that are similar to the one in input.
+
+        When a snippet id is used, the system will return snippets across all documents, that are similar
+        to the one referred to by the given id.
+
+        When a document id is used it will _currently_ search as if you had provided the snippet id of an
+        arbitrary snippet in the document. This means it's for now only meaningful usable with documents which
+        only have one snippet. That will change in the future.
```

:art: just to be sure 😅 
```suggestion
        only have one snippet. That will likely change in the future.
```